### ISSUE TITLE
New UDS Component: Table/Grid

### ISSUE THREAD

**Issue by stdavis** [agrc/kitchen-sink#335]
We need a component to help us with [this issue](https://github.com/agrc/electrofishing-query/issues/184) in electrofishing query.

**Comment by stdavis** [comment]
The client from electrofishing requested the header sorting functionality. It would be great to include this with the first release of this component.

**Comment by stdavis** [comment]
I don't see a PR for this component...

**Comment by steveoh** [comment]
> I don't see a PR for this component...

Yep, you're right. The linked commit from above is now in a draft pr.
